lib/experimental/subject.rb

```diff
@@ -12,8 +12,8 @@ def experiment_bucket(name)
       Experimental.source[name].try { |e| e.in?(self) ? e.bucket(self) : nil }
     end
 
-    def in_bucket?(name, bucket)
-      in_experiment?(name) && experiment_bucket(name) == bucket
+    def in_bucket?(name, bucket, *args)
```

Comment:
I kind of favor the `bucket` param being either single value or array, and internally handling that. I think there is precedent for this in a lot of Ruby standard classes?